pysqldb3/pysqldb3.py

```diff
@@ -1195,133 +1207,36 @@ def xls_to_table(self, input_file=None, sheet_name=0, overwrite=False, schema=No
-                df.to_excel(input_file, index=False, header=True)
-
-                remove_file = True
-                extension = 'xlsx'
-                multi_sheet = False
-            except Exception as e:
-                print("""
-                    Attempt to convert to single-sheet xlsx file failed. 
-                    """)
-                print(e)
-
-                # Restore to original values if conversion fails
-                input_file = old_input_file
-                extension = old_extension
-                remove_file = False
-
-                if extension == 'xlsx':
-                    multi_sheet = len(wb.sheetnames) > 1
-                elif extension == 'xls':
-                    multi_sheet = len(ef.sheet_names) > 1
-
-        # Try bulk input
-        if not multi_sheet and extension == 'xlsx' and not column_type_overrides:
-            # Bulk input
-            success = self._bulk_xlsx_to_table(input_file=input_file, schema=schema, table="stg_{}".format(table))
-
-            # Overwrite if applicable and successful
-            if success and overwrite:
-                self.drop_table(schema=schema, table=table)
-
-            # Move from stg to live
-            if success:
-                self.query("""
-                select * 
-                into {schema}.{table}
-                from {schema}.{stg_table}
-                """.format(schema=schema, table=table, stg_table="stg_{}".format(table)), days=days)
-
-                # Drop stg table
-                self.drop_table(schema=schema, table="stg_{}".format(table))
-
-        # Warn why will not work for xls if bulk wasn't called
-        if multi_sheet or extension != 'xlsx':
+        if multi_sheet:
             print("""
-            Only large, single-sheet xlsx (and csv) files can be loaded quickly via ogr/gdal. 
-            Consider manually converting the file to csv or xlsx.
+                Only the specified sheet (or 1st if not specified will be imported
             """)
+        # if not too big use pandas
 
-        if not success:
-            # Uses the first sheet if no inputted sheet name
-            if extension == 'xlsx':
-                # Grab sheet
-                if sheet_name:
-                    if type(sheet_name) == int and str(sheet_name) not in wb.sheetnames:
-                        sheet_data = list(wb[wb.sheetnames[sheet_name]].values)
-                    else:
-                        sheet_data = list(wb[str(sheet_name)].values)
-                elif not sheet_name:
-                    sheet_data = list(wb.worksheets[0].values)
-
-                # Match previous styles
-                cols = []
-                for c in sheet_data[0]:
-                    if type(c) == str:
-                        cols.append(c.strip().replace(' ', '_').replace('.', '_'))
-                    else:
-                        cols.append(c)
 
-                df = pd.DataFrame(sheet_data[1:], columns=cols)
+        try:
```

Comment:
We can get rid of most of the code here but maybe large tables should still default to _bulk_xlsx_to_table